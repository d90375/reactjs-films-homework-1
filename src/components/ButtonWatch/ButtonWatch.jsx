import React from 'react';

import styles from './ButtonWatch.scss';

const ButtonWatch = () => (
  <button className={styles.buttonWatch} type="button">Watch Now</button>
);

export default ButtonWatch;
