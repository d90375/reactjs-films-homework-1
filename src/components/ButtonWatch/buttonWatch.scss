.buttonWatch {
  width: 150px;
  height: 45px;
  font-size: 20px;
  font-family: "Helvetica Neue", Helvetica, sans-serif;
  color: #fff;
  border-radius: 5px;
  background-color: #0aaee4;
  border: none;
  cursor: pointer;

  &:hover {
    background-color: #0aade4be;
  }

  &:active,
  &:focus {
    outline: none;
  }
}
