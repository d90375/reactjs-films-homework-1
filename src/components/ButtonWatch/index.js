import ButtonWatch from './ButtonWatch';

export default ButtonWatch;
