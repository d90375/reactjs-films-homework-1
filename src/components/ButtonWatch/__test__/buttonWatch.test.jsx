import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import ButtonWatch from '../ButtonWatch';

test('ButtonWatch renders correctly', () => {
  const renderer = new ShallowRenderer();
  renderer.render(<ButtonWatch />);
  const result = renderer.getRenderOutput();
  expect(result).toMatchSnapshot();
});
