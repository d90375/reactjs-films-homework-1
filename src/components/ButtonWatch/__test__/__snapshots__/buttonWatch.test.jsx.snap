// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonWatch renders correctly 1`] = `
<button
  className="buttonWatch"
  type="button"
>
  Watch Now
</button>
`;
